src/main/java/pl/javastart/task/TournamentStats.java

```diff
@@ -1,12 +1,103 @@
 package pl.javastart.task;
 
-import java.util.Scanner;
+import java.util.*;
 
 public class TournamentStats {
 
+    private static final int SORT_BY_FIRST_NAME = 1;
+    private static final int SORT_BY_LAST_NAME = 2;
+    private static final int SORT_BY_RESULT = 3;
+    private static final int SORTING_BY_REVERSE = 2;
+    private static final int SORT_ASCENDING = 1;
+    private static boolean reverseList;
+
     void run(Scanner scanner) {
-        // tutaj dodaj swoje rozwiązanie
-        // użyj przekazanego scannera do wczytywania wartości
+        TournamentStatsWriter tournamentStatsWriter = new TournamentStatsWriter();
```

Comment:
Ten obiekt wykorzystywany jest wyłącznie w `sortListOfPlayers`. Skoro tak, to utwórz go właśnie tam